public/class-gdpr-requests-public.php

```diff
@@ -42,7 +42,7 @@ public function delete_user( $user, $index ) {
 
 		if ( parent::remove_from_requests( $index ) ) {
 			$token = GDPR::generate_pin();
-			GDPR_Email::send( $user->user_email, 'delete-resolved', array( 'token' => $token ) );
+			GDPR_Email::send( $this->get_escaped_user_email_address( $user ), 'delete-resolved', array( 'token' => $token ) );
```

Comment:
```suggestion
                         $email_address = $this->get_escaped_user_email_address( $user )
			GDPR_Email::send( $email_address , 'delete-resolved', array( 'token' => $token ) );
```

Reply:
I just prefer to avoid "train coach" code, feels more readable.

Reply:
Aye, it's a good shout. TBH Jon I'm just trying to do the bare minimum with it to get it to a place where we can replace the existing GDPR plugin with this fork. I don't want to change anything too much, at this stage, just so if anything goes wrong there's less to rule out. 

I will get onto tidying this entire thing up as part of a further PR I have in progress currently.